ballerina/Module.md

```diff
@@ -0,0 +1,6 @@
+Prints "Hello, World!" with a main function.
+[//]: # (above is the module summary)
+
+# Module Overview
+Provides an overview about the module when generating the API documentations.
+For example, refer to https://lib.ballerina.io/ballerina/io/latest
```

Comment:
Shall we update the Module.md and Package.md files, atleast with a short description for now?